client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -305,25 +291,7 @@ const PluginsBrowser = ( { trackPageViews = true, category, search, hideHeader }
 					</div>
 				</FixedNavigationHeader>
 			) }
-			{ isSiteConnected === false && (
-				<Notice
-					icon="notice"
-					showDismiss={ false }
-					status="is-warning"
-					text={ translate( '%(siteName)s cannot be accessed.', {
-						textOnly: true,
-						args: { siteName: selectedSite.title },
-					} ) }
-				>
-					<NoticeAction
-						onClick={ trackSiteDisconnect }
-						href={ `/settings/disconnect-site/${ selectedSite.slug }?type=down` }
-					>
-						{ translate( 'I’d like to fix this now' ) }
-					</NoticeAction>
-				</Notice>
-			) }
-
+			<PluginsNoticeSection />
```

Comment:
Could we rename this to something like `JetpackDisconnectedNotice`?

Reply:
Was thinking the same last night @lsl, will do it before merging